src/EnergyPlus/OutputReportPredefined.cc

```diff
@@ -1341,22 +1341,16 @@ namespace OutputReportPredefined {
         // na
 
         // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
-        int sigDigitCount;
-        std::string stringEntry;
-
         incrementTableEntry(state);
+        int sigDigitCount = 2;
         // check for number of significant digits
         if (present(numSigDigits)) {
             if ((numSigDigits <= 9) && (numSigDigits >= 0)) {
                 sigDigitCount = numSigDigits;
-            } else {
-                sigDigitCount = 2;
             }
-        } else {
-            sigDigitCount = 2;
         }
 
-        if (tableEntryReal < 1e8) { // change from 1e10 for more robust entry writing
+        if (std::abs(tableEntryReal) < 1e8) { // change from 1e10 for more robust entry writing
```

Comment:
Actual change

Reply:
This looks correct. Same change made to 3rd party engine (I just forgot to create an issue here).

Reply:
This makes good sense to me.  I could think of a few cleanups to tack on here (clean up comments, get rid of Objexx Optional argument), but I won't request them here so that I can test and merge this right in.